docker/generate_kafka_pr_template.sh

```diff
@@ -0,0 +1,61 @@
+#!/usr/bin/env bash
```

Comment:
Add license string at the beginning of all new files

Reply:
This has been added